webextension/background.js

```diff
@@ -361,11 +361,13 @@ let window_height = {};
 // due to a race condition with messages.
 function send_to_overlay(id, msg) {
     if (!overlay_ready[id]) {
-	deferred_message[id] = msg;
+	    deferred_message[id] = msg;
+    }
+    else {
```

Comment:
adding `else` to avoid error.
if the overlay is not ready then defer and no need to send message.